private/buf/cmd/buf/buf.go

```diff
@@ -304,6 +305,19 @@ func wrapError(err error) error {
 		return err
 	}
 	if isConnectError {
+		var augmentedConnectError *bufconnect.AugmentedConnectError
+		isAugmentedConnectErr := errors.As(err, &augmentedConnectError)
```

Comment:
When will this ever be false in this context, if ever?

Reply:
Don't know the `buf` code well enough to be sure, but if we were returning a `connect.Error` from anywhere that _wasn't_ wrapped in the interceptor (probably not? — but will that always be true?).

Just trying to be cautious, since this is more of a best-effort hint to users that they may be in this situation, rather than an iron-clad "we know you are definitely in this situation."